index.js

```diff
@@ -503,7 +503,21 @@ function getRedirectUrl (url) {
   }
 }
 
-module.exports = fp(fastifyStatic, {
+const fastifyStatic = fp(plugin, {
```

Comment:
You do not need to change this file. It is done inside `fastify-plugin`.

Reply:
Wdym? I just followed [this](https://github.com/fastify/fastify-cookie/pull/97) PR to add named export for consistency

Reply:
It is something already done by `fastify-plugin`. 
That means this package already have named export.

Reply:
Oh well, so fastify-cookie has legacy implementation, it's clear for me now. Fixed